client/landing/subscriptions/components/subscription-manager-context/index.tsx

```diff
@@ -1,33 +1,42 @@
-import { createContext, useContext } from '@wordpress/element';
+import { createContext, useContext } from 'react';
 
-export const SubscriptionsPortal = 'subscriptions';
-export const ReaderPortal = 'reader';
-export type Portal = typeof SubscriptionsPortal | typeof ReaderPortal;
+export enum SiteSubscriptionsPortal {
```

Comment:
I wouldn't prefix it with `Site` as it isn't as all-encompassing as eventually Comments and Settings will be brought to Reader.

Reply:
Ops, you are right. Thanks for catching this out.